tests/Mvc/Cli/Commands/Data/RestoreCommandPathTraversalTest.php

```
@@ -0,0 +1,306 @@
+
+		$this->assertTrue( $result['valid'], 'Special characters in filename should be allowed if within base' );
+	}
+
+	// Helper methods
+
+	private function createRestoreCommand(): RestoreCommand
+	{
+		return new RestoreCommand();
+	}
+
+	private function simulatePathValidation( RestoreCommand $command, string $basePath, string $inputPath ): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$command'. (undefined)

(UnusedFormalParameter)